Sources/BraintreePayPal/BTPayPalRequest.swift

```diff
@@ -96,6 +96,10 @@ import BraintreeCore
     /// :nodoc: Exposed publicly for use by PayPal Native Checkout module. This property is not covered by semantic versioning.
     @_documentation(visibility: private)
     public var paymentType: BTPayPalPaymentType
+    
+    /// Optional: A user's phone number to initiate a quicker authentication flow in the scenario where the user has a PayPal account
+    /// identified with the same phone number.
+    public var userPhoneNumber: String?
```

Comment:
It looks like Android has a separate phone object for this: https://github.com/braintree/braintree_android/pull/1186. Are we able to take the same approach on iOS so the interface is the same?